### PR TITLE
Update `getting-started.md`

### PR DESCRIPTION
**Pull request by ParadoxV5**
* Fix the first sample code
* Fix grammar and formatting

### CODE REVIEW

File: guides/src/getting-started/getting-started.md [hunk 1]
```diff
@@ -137,15 +137,15 @@ country = %w(US JP UK).sample() or raise
...
+The change lets the type checking succeed.
 
 ```
 $ steep check
 # Type checking files:
 
 .........................................................
 
-No type error detected. 🧉
+No type error detected. 🫖
```
Comment (ParadoxV5) [inline guides/src/getting-started/getting-started.md:148]: Original auther rolled 1 of 3 in 11 🎲

File: guides/src/getting-started/getting-started.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Type the following Ruby code in your editor, and save it as `lib/hello.rb`.
 currencies = { US: "$", JP: "¥", UK: "£" }
 country = %w(US JP UK).sample()
 
-puts "Hello! The price is #{currencies[country.to_sym]}100. 💸"
+puts "Hello! The price is #{currencies[country]}100. 💸"
```
Comment (ParadoxV5) [inline guides/src/getting-started/getting-started.md:57]: The mistake in question

File: guides/src/getting-started/getting-started.md [hunk 3]
```diff
@@ -137,15 +137,15 @@ country = %w(US JP UK).sample() or raise
...
-The change let the type checking succeed.
+The change lets the type checking succeed.
 
 ```
 $ steep check
 # Type checking files:
 
 .........................................................
 
-No type error detected. 🧉
```
Comment (ParadoxV5) [inline guides/src/getting-started/getting-started.md:148]: 1 of 3 in 11 🎲

File: guides/src/getting-started/getting-started.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ Type the following Ruby code in your editor, and save it as `lib/hello.rb`.
 currencies = { US: "$", JP: "¥", UK: "£" }
 country = %w(US JP UK).sample()
 
-puts "Hello! The price is #{currencies[country.to_sym]}100. 💸"
```
Comment (ParadoxV5) [inline guides/src/getting-started/getting-started.md:57]: The mistake in question

Comment (soutaro) [reply guides/src/getting-started/getting-started.md:57]: `country` is a `String` (or `nil`), not `Symbol`.

**Review by soutaro — APPROVED** [review]
Thanks!

**Pull request merged**